### PR TITLE
Bug 2041546: ovn-kubernetes: set RAFT election timer at RAFT cluster creation time

### PR DESCRIPTION
**Pull request by dcbw**
If supported by OVN, which was added in 21.12 and later.

### CODE REVIEW

**Comment by dcbw** [comment]
/retest
https://bugzilla.redhat.com/show_bug.cgi?id=2020189

File: bindata/network/ovn-kubernetes/ovnkube-master.yaml [hunk 1]
```diff
@@ -248,8 +248,15 @@ spec:
...
               if [[ "${K8S_NODE_IP}" == "${CLUSTER_INITIATOR_IP}" ]]; then
+                # set DB election timer at DB creation time if OVN supports it
+                election_timer=
+                if test -n "$(/usr/share/ovn/scripts/ovn-ctl --help 2>&1 | grep "\--db-nb-election-timer")"; then
+                  election_timer="--db-nb-election-timer=$(({{.OVN_NB_RAFT_ELECTION_TIMER}}*1000))"
+                fi
+
                 exec /usr/share/ovn/scripts/ovn-ctl ${OVN_ARGS} \
                 --ovn-nb-log="-vconsole:${OVN_LOG_LEVEL} -vfile:off -vPATTERN:console:{{.OVN_LOG_PATTERN_CONSOLE}}" \
+                ${election_timer} \
```
Comment (tssurya) [inline bindata/network/ovn-kubernetes/ovnkube-master.yaml:259]: so after we set it here  on startup, what happens here: https://github.com/openshift/ovn-kubernetes/blob/c679b0c662b4386e4da9e6be78f4eeeed7157f40/go-controller/pkg/ovndbmanager/ovndbmanager.go#L111 and  https://github.com/openshift/ovn-kubernetes/blob/c679b0c662b4386e4da9e6be78f4eeeed7157f40/go-controller/pkg/ovndbmanager/ovndbmanager.go#L316? I am a but confused of what db checker is doing...

Comment (tssurya) [reply bindata/network/ovn-kubernetes/ovnkube-master.yaml:259]: oh Dan explains it https://bugzilla.redhat.com/show_bug.cgi?id=2041546#c0

**Review by tssurya — APPROVED** [review]
/lgtm

**Comment by dcbw** [comment]
/override ci/prow/e2e-agnostic-upgrade
this is an SDN job that is in no way affected by this ovnkube PR

**Pull request merged**